<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="PC2017000016" ignored="0" name="Test3" tests="1" failures="0" timestamp="17 Jan 2018 12:52:38 GMT" time="25.069" errors="0">
  <testcase name="@BeforeMethod Befour" time="25.026" classname="NERL.Pledgee_Initiation_Main">
    <failure type="org.openqa.selenium.WebDriverException" message="Error forwarding the new session Error forwarding the request Connect to 172.29.22.42:5566 [/172.29.22.42] failed: Connection timed out: connect
Command duration or timeout: 21.01 seconds">
      <![CDATA[org.openqa.selenium.WebDriverException: Error forwarding the new session Error forwarding the request Connect to 172.29.22.42:5566 [/172.29.22.42] failed: Connection timed out: connect
Command duration or timeout: 21.01 seconds
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:215)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:167)
	at org.openqa.selenium.remote.JsonWireProtocolResponse.lambda$new$0(JsonWireProtocolResponse.java:53)
	at org.openqa.selenium.remote.JsonWireProtocolResponse.lambda$getResponseFunction$2(JsonWireProtocolResponse.java:91)
	at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$22(ProtocolHandshake.java:365)
	at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)
	at java.util.Spliterators$ArraySpliterator.tryAdvance(Unknown Source)
	at java.util.stream.ReferencePipeline.forEachWithCancel(Unknown Source)
	at java.util.stream.AbstractPipeline.copyIntoWithCancel(Unknown Source)
	at java.util.stream.AbstractPipeline.copyInto(Unknown Source)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
	at java.util.stream.FindOps$FindOp.evaluateSequential(Unknown Source)
	at java.util.stream.AbstractPipeline.evaluate(Unknown Source)
	at java.util.stream.ReferencePipeline.findFirst(Unknown Source)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:368)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:159)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:637)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:250)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:236)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:137)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:174)
	at NERL.TestFFonGrid.Grid(TestFFonGrid.java:32)
	at NERL.Pledgee_Initiation_Main.Befour(Pledgee_Initiation_Main.java:38)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.openqa.selenium.WebDriverException: Error forwarding the new session Error forwarding the request Connect to 172.29.22.42:5566 [/172.29.22.42] failed: Connection timed out: connect
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'PC2017000016', ip: '172.29.22.45', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_131'
Driver info: driver.version: unknown
	at org.openqa.grid.web.servlet.handler.RequestHandler.process(RequestHandler.java:115)
	at org.openqa.grid.web.servlet.DriverServlet.process(DriverServlet.java:83)
	at org.openqa.grid.web.servlet.DriverServlet.doPost(DriverServlet.java:67)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
	at org.seleniumhq.jetty9.servlet.ServletHolder.handle(ServletHolder.java:808)
	at org.seleniumhq.jetty9.servlet.ServletHandler.doHandle(ServletHandler.java:587)
	at org.seleniumhq.jetty9.server.session.SessionHandler.doHandle(SessionHandler.java:221)
	at org.seleniumhq.jetty9.server.handler.ContextHandler.doHandle(ContextHandler.java:1127)
	at org.seleniumhq.jetty9.servlet.ServletHandler.doScope(ServletHandler.java:515)
	at org.seleniumhq.jetty9.server.session.SessionHandler.doScope(SessionHandler.java:185)
	at org.seleniumhq.jetty9.server.handler.ContextHandler.doScope(ContextHandler.java:1061)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
	at org.seleniumhq.jetty9.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97)
	at org.seleniumhq.jetty9.server.Server.handle(Server.java:499)
	at org.seleniumhq.jetty9.server.HttpChannel.handle(HttpChannel.java:310)
	at org.seleniumhq.jetty9.server.HttpConnection.onFillable(HttpConnection.java:257)
	at org.seleniumhq.jetty9.io.AbstractConnection$2.run(AbstractConnection.java:540)
	at org.seleniumhq.jetty9.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:635)
	at org.seleniumhq.jetty9.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:555)
	... 1 more
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod Befour -->
  <testcase name="@AfterMethod After" time="0.0" classname="NERL.Pledgee_Initiation_Main">
    <skipped/>
  </testcase> <!-- @AfterMethod After -->
  <testcase name="Pledgee_Initiation_Submit" time="0.0" classname="NERL.Pledgee_Initiation_Main">
    <skipped/>
  </testcase> <!-- Pledgee_Initiation_Submit -->
</testsuite> <!-- Test3 -->
